Makefile

```diff
@@ -56,6 +56,9 @@ CLIENT_PACKAGES = $(GO_PROJECT)/cmd/rookctl
 # server projects that we build on server platforms
 SERVER_PACKAGES = $(GO_PROJECT)/cmd/rook
 
+# Flexvolume projects that we build on server platforms
+FLEXVOLUME_PACKAGES = $(GO_PROJECT)/cmd/rookflexvolume
```

Comment:
just add the rookflexvolume to `SERVER_PACKAGES`. no need for FLEXVOLUME_PACKAGES